apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.test.js

```diff
@@ -89,6 +89,86 @@ describe('AllResidentBillingReceiptsService', () => {
                 expect(receipt.services).not.toBeNull()
             })
         })
+
+        it('returns file', async () => {
+            const userClient = await makeClientWithProperty()
+            const support = await makeClientWithSupportUser()
+            const adminClient = await makeLoggedInAdminClient()
+
+            const [integration] = await createTestBillingIntegration(support)
+            const [billingContext] = await createTestBillingIntegrationOrganizationContext(userClient, userClient.organization, integration)
+
+            const integrationClient = await makeClientWithServiceUser()
+            await createTestBillingIntegrationAccessRight(support, integration, integrationClient.user)
+            const [billingProperty] = await createTestBillingProperty(integrationClient, billingContext, {
+                address: userClient.property.address,
+            })
+            const [billingAccount, billingAccountAttrs] = await createTestBillingAccount(integrationClient, billingContext, billingProperty)
+
+            const residentUser = await makeClientWithResidentUser()
+            const [resident] = await registerResidentByTestClient(residentUser, {
+                address: userClient.property.address,
+                addressMeta: userClient.property.addressMeta,
+                unitName: billingAccountAttrs.unitName,
+            })
+            await registerServiceConsumerByTestClient(residentUser, {
+                residentId: resident.id,
+                accountNumber: billingAccountAttrs.number,
+                organizationId: userClient.organization.id,
+            })
+            const [receipt] = await createTestBillingReceipt(integrationClient, billingContext, billingProperty, billingAccount)
+
+            // create BillingReceiptFile
+            const [receiptFile] = await createTestBillingReceiptFile(adminClient, receipt, billingContext)
+
+            // set primary file
+            await updateTestBillingReceipt(integrationClient, receipt.id, {
+                file: { connect: { id: receiptFile.id } },
+            })
+
+            // getting the result
+            const residentBillingReceipts = await ResidentBillingReceipt.getAll(residentUser)
+
+            expect(residentBillingReceipts).toBeDefined()
+            expect(residentBillingReceipts).not.toHaveLength(0)
```

Comment:
Is the exact number not known?

Reply:
Looks like not, since this is an exact copy of another test with a little changed set up